lua_modules/liquidcrystal/NTest_lc_i2c4bit.lua

```diff
@@ -0,0 +1,116 @@
+-- Run LiquidCrystal through some basic tests
```

Comment:
Do we really want to place Lua-module tests beside the module and C-module tests in /tests?

Reply:
I think it's cleaner, but I admit it differs from the rest of our tree.  (I think I'd rather have per-feature directories with their dependencies (e.g., `app/mqtt/lib/*`, `app/mqtt/mqtt-lua.c`) rather than one big amalgamated `app/modules` and random other directories in `app/`, with exceptions being made for things like `LwIP` that are cross-cutting.... but it's not my tree.)

If it'd make you happier, I can move this into `/tests`, probably as `NTest_liquidcrystal_i2c4bit` rather than `..._lc_...`.

Reply:
Its not about me being happy (well I still like to be happy) but about what makes sense. As we have a docs tree we might also have a tests tree, since it is easier to collect all the tests then. But the reason for the docs tree seems to be a technical one (not sure about that). 
From a developer perspective it would make sense to have the test with the code. At $work however we also have an extra test project sitting besides the code project.
So I think your proposal above is fine.
The filename can't get longer though. (31 chars already if I didn't miscount)